template.rb

```diff
@@ -4,7 +4,8 @@
 run 'bundle install'
 git :init
 git add: "."
-git commit: "-a -m 'Bare Rails application\nGenerated using https://github.com/alphagov/govuk-rails-app-template'"
+command = "#{File.basename($0)} #{ARGV.join(' ')}"
```

Comment:
Which parts of the environment were you thinking of?

There's no easy way that I can think of to capture only the ENV vars set on the command line...